test-suite/coq-makefile/timing/per-file-after/A.v.timing.diff.desired

```diff
@@ -1,9 +1,9 @@
-    After | Code                                                |    Before ||    Change |  % Change
+After     | Code                                                | Before    || Change    | % Change 
 ----------------------------------------------------------------------------------------------------
- 0m14.06s | Total                                               |  0m00.72s || +0m13.34s | +1854.02%
+0m13.89s  | Total                                               | 0m00.70s  || +0m13.18s | +1859.66%
 ----------------------------------------------------------------------------------------------------
-0m13.582s | Chars 163 - 208 [Definition~foo1~:=~Eval~comp~i...] | 0m00.353s || +0m13.22s | +3747.59%
-0m00.335s | Chars 069 - 162 [Definition~foo0~:=~Eval~comp~i...] | 0m00.225s || +0m00.11s |   +48.88%
-0m00.152s | Chars 000 - 026 [Require~Coq.ZArith.BinInt.]        | 0m00.142s || +0m00.01s |    +7.04%
-   0m00.s | Chars 027 - 068 [Declare~Reduction~comp~:=~nati...] |    N/A    || +0m00.00s |    N/A   
-   N/A    | Chars 027 - 068 [Declare~Reduction~comp~:=~vm_c...] |    0m00.s || +0m00.00s |    N/A   
\ No newline at end of file
+0m13.477s | Chars 163 - 208 [Definition~foo1~:=~Eval~comp~i...] | 0m00.349s || +0m13.12s | +3761.60%
+0m00.277s | Chars 069 - 162 [Definition~foo0~:=~Eval~comp~i...] | 0m00.221s || +0m00.05s | +25.33%  
+0m00.14s  | Chars 000 - 026 [Require~Coq.ZArith.BinInt.]        | 0m00.139s || +0m00.00s | +0.71%   
+0m00.s    | Chars 027 - 068 [Declare~Reduction~comp~:=~nati...] |    N/A    || +0m00.00s | N/A      
+   N/A    | Chars 027 - 068 [Declare~Reduction~comp~:=~vm_c...] | 0m00.s    || +0m00.00s | N/A      
```

Comment:
Github says this one still misses an end of file newline, is that normal?

Reply:
Very strange, let me look into this

Reply:
Should be fixed now

Reply:
So what was going on?

Reply:
I believe this test-suite-test was insensitive to lack of trailing newlines, and I was picking up the wrong version of the script when regenerating it, I believe